packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -35,6 +38,22 @@ async fn main() -> Result<(), GraphError> {
             err
         })?;
 
+    #[cfg(debug_assertions)]
+    let account_id = AccountId::new(Uuid::nil());
+    #[cfg(not(debug_assertions))]
+    let account_id = AccountId::new(Uuid::new_v4());
```

Comment:
While I do think this is a neat way to do this, I don't think we want to even create another UUID in release mode, and rather just use `Uuid::nil()` in either cases. This way we can still run our manual tests against release mode without having to change the UUID.